<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>WinkStudio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/images/images.qrc">
    <normaloff>:/images/wink.png</normaloff>:/images/wink.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btn_screenrecorder">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>屏幕录制</string>
          </property>
          <property name="icon">
           <iconset resource="../../resource/images/images.qrc">
            <normaloff>:/images/camera.svg</normaloff>:/images/camera.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_audioplayer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>音乐播放器</string>
          </property>
          <property name="icon">
           <iconset resource="../../resource/images/images.qrc">
            <normaloff>:/images/music.svg</normaloff>:/images/music.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_imageformat">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>图片格式转换</string>
          </property>
          <property name="icon">
           <iconset resource="../../resource/images/images.qrc">
            <normaloff>:/images/image.svg</normaloff>:/images/image.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QStackedWidget" name="stackedwidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_set">
    <property name="title">
     <string>设置</string>
    </property>
    <property name="icon">
     <iconset resource="../../resource/images/images.qrc">
      <normaloff>:/images/general.svg</normaloff>:/images/general.svg</iconset>
    </property>
    <addaction name="action_set"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助</string>
    </property>
    <property name="icon">
     <iconset resource="../../resource/images/images.qrc">
      <normaloff>:/images/globe.svg</normaloff>:/images/globe.svg</iconset>
    </property>
    <addaction name="action_feedback"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_theme">
    <property name="title">
     <string>主题</string>
    </property>
    <property name="icon">
     <iconset resource="../../resource/images/images.qrc">
      <normaloff>:/images/skin.svg</normaloff>:/images/skin.svg</iconset>
    </property>
    <addaction name="action_dark"/>
    <addaction name="action_light"/>
   </widget>
   <addaction name="menu_set"/>
   <addaction name="menu_help"/>
   <addaction name="menu_theme"/>
  </widget>
  <action name="action_set">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="action_feedback">
   <property name="text">
    <string>反馈</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_dark">
   <property name="text">
    <string>dark</string>
   </property>
  </action>
  <action name="action_light">
   <property name="text">
    <string>light</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
